Refactor instructions to run backend

manuhortet opened the pull request:
This aims to simplify the process of setting up and running the project locally.

- Minor refactor in the instructions to set the project locally.
- Minor refactor in the `package.json`

brortao commented on package.json:
```diff
@@ -8,8 +8,9 @@
     "yarn": "1.x"
   },
   "scripts": {
+    "prepare": "yarn install && cp example.env .env && yarn codegen",  
```
nice!

brortao commented on package.json:
```diff
@@ -8,8 +8,9 @@
+    "prepare": "yarn install && cp example.env .env && yarn codegen",  
     "start": "node dist/index.js",
-    "dev": "dotenv -- nodemon -e ts,graphql -x ts-node src/index.ts",
+    "dev": "yarn prisma deploy && dotenv -- nodemon -e ts,graphql -x ts-node src/index.ts",
```
`prisma deploy` probably shouldn't be in a script called `dev`, since it can affect remote (i.e. staging or prod) environment. it just depends on the values in your `.env`.

manuhortet commented:
@King-Mob It'd be nice to update our [dev environment setup](https://wiki.wobbly.app/index.php/Development_environment_setup) wiki page. I can write it. On the meantime I added a link to our discord in case people need help.

@brortao 
- Calling `yarn install` inside `yarn prepare` was provoking a recursive situation when calling `yarn install`, so I took it out. 
- I moved our `example.env` to `env/local.env`. The working env is still the one defined in `.env` at root level.
